File: src/test/java/android/os/Bundle.java
package android.os;

import java.util.HashMap;
import java.util.Map;

public class Bundle {
    Map<String, Object> map;

    public Bundle() {
        map = new HashMap<>();
    }

    public Object get(String key) {
        return map.get(key);
    }

    public void putString(String key, String value) {
        map.put(key, value);
    }

    public void putDouble(String key, double value) {
        map.put(key, value);
    }

    public void putParcelableArray(String key, Parcelable[] value) {
        map.put(key, value);
    }

    public double getDouble(String key) {
        return (double)map.get(key);
    }

    public String getString(String key) {
        return (String)map.get(key);
    }

    public int size() {
        return map.keySet().size();
    }
}

File: src/test/kotlin/android/os/Bundle.kt
package android.os


class Bundle:Parcelable {

    var map= HashMap<String, Any>()
    operator fun get(key: String): Any? {
        return map[key]
    }

    fun putString(key: String, value: String) {
        map[key] = value
    }

    fun putDouble(key: String, value: Double) {
        map[key] = value
    }

    fun putLong(key: String, value: Long) {
        map[key] = value
    }

    fun putInt(key: String, value: Int) {
        map[key] = value
    }

    fun putParcelableArray(key: String, value: Array<Parcelable?>) {
        map[key] = value
    }

    fun getDouble(key: String): Double {
        return map[key] as Double
    }

    fun getString(key: String): String? {
        return map[key] as String?
    }

    fun size(): Int {
        return map.keys.size
    }

    override fun describeContents(): Int = 0
    override fun writeToParcel(dest: Parcel?, flags: Int) {}
}
